### ISSUE TITLE
Example configuration file doesn't work "out of the box"

### ISSUE THREAD

**Issue by blthayer** [FSEC-Photovoltaics/pvrpm-lcoe#44]
As a starting place for new users, the example configuration file should be configured in such a way that it "just works."

When I attempted to use it, I received the following error: `Error loading and verifying case: Number of monitoring modes for component 'module' must be 1 or equal to the number of failures`.

I had to adjust the `monitoring` under `module` to comment out one of the monitoring distributions to get it to work:

```yaml
  monitoring:
    normal_monitoring: # this key name can be anything you want
      distribution: exponential
      parameters:
        mean: 5

#    defective_monitoring:
#      distribution: normal
#      parameters:
#        mean: 15
#        std: 5

```

**Comment by SarthakJariwala** [comment]
I second this.
As a new user, I would expect the example configuration file to "just work."

**Comment by brandons209** [comment]
Fixed in a098c23e4f051e1a9f7099464984b2ec3331c258
